### ISSUE TITLE
Using Tor version 2.41 help

### ISSUE THREAD

**Issue by Sthomasnl** [doged/electrum-doged#29]
I have the legacy version of the 2.4.1 Electrum-XVG on an older PC that I just got working again to do some backup so I can move to my new PC.  When connecting with the old version it does not connect and most likely this is because the Tor version is being used now, howeverm, after downloading the 64b Tor version for Windows it says it does not see a wallet.  The recovery seed (which matches perfectly with the words and order shown in the old version) does not allow the "next" button to light up.  I would prefer to restore with the XVG token I have obviously.  Is there a way to browse to the file in the old version directory so it finds it? At this point not sure what else to try.  The old version doesn't connect, and the new version doesn't appear to acknowledge the seed recovery words?  Any help is appreciated :)

**Comment by justinvforvendetta** [comment]
legacy version of dogecoindark electrum? or verge? the old verge electrum should work with the new electrum. you can try copying the contents of the data folder over.

**Comment by Sthomasnl** [comment]
Electrum.
It’s labeled Electrum-XVG.
When it opens up it shows the history as “unverified”
And under the tools and network settings it says not connected,
disconnected from the server. It is currently trying to connect with IP
142.93.6.38:50002

I have attempted to try E1.vergedashelectrum.com and the others in the
list, but it does not connect and shows a red button in the bottom right.
This is of course using the old version of the client which I suspect is
the main issue.

When downloading the newer tor client x64 zip file and running electrum-xvg
from that new file location it opens an electrum install wizard, Giving me
choices to create a new wallet or restore/import keys. I could create a new
wallet but that doesn't help me i dont think. When i tried the seed words
in the same order as they show from the old version the next button is
grayed out as if its not correct or it needs something additional but its
correct.

Anyway, Thanks for your help and quick response.

**Comment by justinvforvendetta** [comment]
ive never heard of those servers, we discontinued clearnet electrum 2.4 support years ago. my advice would be to go into your addresses, export the private keys, and import them into the newest electrum.

**Comment by Sthomasnl** [comment]
Thanks for the tip!

I think I may be using the wrong files perhaps for the new wallet to be
honest.  Both of them are identical - they say "Electrum-XVG 2.4.1" despite
the zip being what I thought was the correct new client.  The default
servers however in the "new" download show two addresses that end in
".Onion" with port 50002.  gozwhpes24mibk7.onion & qz6pnufyqjexudq6.onion
Does this sound right?

If not can you post the DL location for the correct wallet for XVG and ill
try that.  Perhaps I am using the incorrect old client just packaged up in
a different zip.

**Comment by Sthomasnl** [comment]
Sunerok,
I got it going.  I just figured it out.  I opened a ticket on the
VergeCurrency.com website but at this point it can be closed.

How I resolved the issue was I replaced the Network.Py data from the site
to my local files.  Restarted the Tor Electrum wallet, and it connected.
I was then able to restore the seed successfully which made it easier for
me since I was thinking I might have to restore private keys but that ended
up not being needed.  Thanks for the help and sorry to be a pain :)
